operator/Makefile

```diff
@@ -76,20 +76,23 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: test
-test: manifests envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./test/... -v  -timeout 30m -ginkgo.skip-file="e2e"
```

Comment:
Why did you remove this here?

Reply:
Because that's essentially to execute component tests, which we have a separate step for now.